conf/modules.config

```diff
@@ -635,14 +635,37 @@ process{
         ]
     }
 
-    withName: 'GENOTYPEGVCFS' {
+    withName: 'GATK4_GENOMICSDBIMPORT' {
+        ext.prefix       = { meta.num_intervals > 1 ? meta.id : "joint_interval" }
         ext.when         = { params.tools && params.tools.contains('haplotypecaller') && params.joint_germline}
         publishDir       = [
+            enabled: false,
             mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/${meta.id}/haplotypecaller"},
+            path: { "${params.outdir}/variant_calling/${ext.prefix}/genomicsDB"},
+            pattern: { "${ext.prefix}/" }
+        ]
+    }
+
+    withName: 'GATK4_GENOTYPEGVCFS' {
+        ext.when         = { params.tools && params.tools.contains('haplotypecaller') && params.joint_germline}
+        ext.prefix       = { meta.num_intervals > 1 ? meta.id: "joint_interval" }
+        publishDir       = [
+            enabled: params.no_intervals,
```

Comment:
this should probably be more related to whether there is < 2 intervals. In this case we must use `saveAs`